### ISSUE TITLE
Recompilation of LimitedCounterIncFormal would cause an infinite compiling.

### ISSUE THREAD

**Issue by Readon** [SpinalHDL/SpinalHDL#703]
When I recompile by 
``` shell
sbt "tester/runMain spinal.tester.code.LimitedCounterIncFormal"
```
without clean on Windows.
The sbt would stuck into infinite loop of displaying "[info] Compiling 2 Scala sources to xxxxxx\SpinalHDL\tester\target\scala-2.11\classes ..."

Which makes clean required in each compling, which take too much time.

**Comment by Readon** [comment]
I find that it's about recompile calss in "tester" subproject.

**Comment by Dolu1990** [comment]
Weird XD
Could you try with SpinalHDL 2.12 / 2.13 if it also apply ?

**Comment by Readon** [comment]
the time can be reduced by "tester/clean; tester/runMain xxxx".



---Original---
From: ***@***.***&gt;
Date: Sat, Apr 30, 2022 23:49 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [SpinalHDL/SpinalHDL] Recompilation of LimitedCounterIncFormal would cause an infinite compiling. (Issue #703)




 
Weird XD
 Could you try with SpinalHDL 2.12 / 2.13 if it also apply ?

**Comment by Readon** [comment]
i did not meet this recently.
I think we can close it now.



---Original---
From: ***@***.***&gt;
Date: Wed, Nov 16, 2022 20:30 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [SpinalHDL/SpinalHDL] Recompilation of LimitedCounterIncFormalwould cause an infinite compiling. (Issue #703)




 
@Readon any updates? I can't reproduce this issue on Linux.

**Comment by Readon** [comment]
as a note, if I cleaned the tester sub project by sbt "tester/clean", then it would work.



---Original---
From: ***@***.***&gt;
Date: Wed, Nov 16, 2022 20:35 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [SpinalHDL/SpinalHDL] Recompilation of LimitedCounterIncFormalwould cause an infinite compiling. (Issue #703)




 
Closed #703 as completed.
